usr/share/rear/lib/global-functions.sh

```diff
@@ -632,11 +633,13 @@ umount_mountpoint() {
     Log "Forced unmount of '$mountpoint'"
     umount $v -f $mountpoint >&2
     if [[ $? -eq 0 ]] ; then
-        return 0
+        echo 0
```

Comment:
Hello @gozora 
You're right, of course. My apologies for missing this. I looked for calls to `umount_mountpoint()` throughout the scripts, but missed the fact that it could also be called by other functions.
I'll revert this change and adapt my own code accordingly.